src/index.ts

```diff
@@ -92,3 +92,21 @@ export const addKeypairToEnvFile = async (
   const secretKeyString = keypairToSecretKeyJSON(keypair);
   await appendFile(fileName, `\n${variableName}=${secretKeyString}`);
 };
+
+export const requestAndConfirmAirdrop = async (
+  connection: Connection,
+  publicKey: PublicKey,
+  amount: number,
+) => {
+  let airdropTransactionSignature = await connection.requestAirdrop(
+    publicKey,
+    amount,
+  );
+  // Wait for airdrop confirmation
+  const latestBlockHash = await connection.getLatestBlockhash();
+  await connection.confirmTransaction({
```

Comment:
I think we should return the promise
```suggestion
  return await connection.confirmTransaction({
```

Reply:
Excellent idea, people might want to check aspects of the confirmed transaction!